repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -475,44 +486,55 @@ def map_repositories(packages):
 
     if repo_without_mapping:
         report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping)
+                                repo_without_mapping,
+                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of"
+                                "the Red Hat Enterprise Linux 7 product.")
```

Comment:
Missing whitespace causing:
_component ofthe Red Hat Enterprise Linux 7 product._

Reply:
Solved in #551